lean/src/test.lean

```diff
@@ -1046,10 +1051,12 @@ theorem mathd_numbertheory_135
   (n a b c: ℕ)
   (h₀ : n = 3^17 + 3^10)
   (h₁ : 11 ∣ (n + 1))
-  (h₂ : odd a ∧ odd c)
-  (h₃ : ¬ 3 ∣ b)
-  (h₄ : n = 10*(10*(10*(10*(10*(10*(10*(10*a +b) +c) +a) +c) +c) +b) +a) +b) :
-  10*(10 * a + b) + c = 129 :=
+  (h₂ : [a,b,c].pairwise(≠))
```

Comment:
Question: the pairwise constraint is added for `h₃` or it's stated in the original statement?

Reply:
It is in the original statement: "where $A,B,C$ are distinct digits such that ..."